kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -606,6 +604,7 @@ def build(self):
         #     by a margin of at least min_to_pad_fr
         # (2) a multiple of all subsampling factors of frequential scattering:
         #     2**1, 2**2, etc. up to 2**K_fr = (2**J_fr / 2**oversampling_fr)
+        N_input_fr = (self.J+1) * self.Q[0]
```

Comment:
is this accurate?

Reply:
not exactly but it doesn't need to be exact. this is only needed for
```python
N_padded_fr_subsampled = (N_input_fr + min_to_pad_fr) // (2 ** K_fr)
```

so it's OK to be off by ± 2**(K_fr-1)

@cyrusvahidi if you know how to get the exact number, let me know! i'll replace

Reply:
bon i see now.